scripts/ios-configure-glog.sh

```diff
@@ -28,6 +28,10 @@ if [ -h "test-driver" ]; then
     rm test-driver
 fi
 
+# Manually disable gflags include.
```

Comment:
Since this is a bit hacky fix, can you please add a link to the issue to the comment to ensure the context is shared?

Reply:
Done.